lab2.html

```diff
@@ -4,22 +4,29 @@
 </head>
 <body>
   <h1> About Michelle </h1>
+    <p id= "greeting">  </p>
   <ul>
-    <li> stuff </li>
-    <li> more stuff </li>
-<h2> What do you know about Michelle? </h2>
+    <h2> Bio </h2>
```

Comment:
Don't need space between the HTML tags and your content, that will actually cause HTML to artificially inject spaces into your content.